sixtyfps_runtime/interpreter/api.rs

```diff
@@ -152,7 +152,7 @@ impl PartialEq for Value {
             Value::Bool(lhs) => matches!(other, Value::Bool(rhs) if lhs == rhs),
             Value::Image(lhs) => matches!(other, Value::Image(rhs) if lhs == rhs),
             Value::Array(lhs) => matches!(other, Value::Array(rhs) if lhs == rhs),
-            Value::Model(lhs) => matches!(other, Value::Model(rhs) if Rc::ptr_eq(lhs, rhs)),
+            Value::Model(lhs) => matches!(other, Value::Model(rhs) if lhs == rhs),
```

Comment:
I actually wonder if this was not the only reason we implemented PartialEq manually.
We could just use #[derive(PartialEq)] instead, now.

Reply:
I can do that, but would prefer to do that in a separate patch.